webservices/args.py

```diff
@@ -80,12 +80,13 @@ class IndexValidator(OptionValidator):
 
     :param Base model: SQLALchemy model.
     :param list exclude: Optional list of columns to exclude.
+    :param list extra: Optional list of extra columns to include.
     """
-    def __init__(self, model, extra=None, exclude=None, schema=None):
+    def __init__(self, model, extra=None, exclude=None):
         self.model = model
+        self.schema = self.model.__table__.schema
```

Comment:
We can get rid of this instance variable and use `self.model.__table__.schema` on line 102 below, as we are anyway using `self.model.__tablename__`.